incubator/eventrouter/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
This file was taken directly from one of the example Charts in the docs (I forget which one).  If this isn't necessary, I'm happy to remove